Website/Packages/datax-pipeline/src/modules/flowDefinition/flowReducer.js

```diff
@@ -20,8 +20,8 @@ const INITIAL_FLOW_STATE = {
     scale: {
         jobNumExecutors: '4',
         jobExecutorMemory: '1000',
-        jobDatabricksAutoScale: true,
-        jobDatabricksMinWorkers: '1',
+        jobDatabricksAutoScale: false,
+        jobDatabricksMinWorkers: '3',
```

Comment:
should this be 3 ?

Reply:
3 is the new default value. Users can still move the slider the on the UI to choose a different value